### ISSUE TITLE
Only gets status once

### ISSUE THREAD

**Issue by PhilT** [demantz/WearNetworkNotifications#1]
I've just installed this on my OnePlus One and Sony SmartWatch 3. There was no signal coming through so I restarted both devices and got 3 bars of LTE which is what the phone reported once the watch app had booted up. However, it never changed after that.

Happy to help debug as I'd really like this to work as I get a choppy signal on my work to work on the train.

Just let me know what you need.

Cheers,
Phil

**Comment by demantz** [comment]
Hi Phil,

could you send me the logs of both the watch and the phone. Just open the app in the phone, hit the menu button and press 'show handheld log'. It opens a dialog that offers you to share the log via email. Do it again with 'show wearable log'. My email is Dennis.mantz@googlemail.com but I will be on vacation for two weeks by now. You'll have to be patient, sorry!

Cheers Dennis

**Comment by PhilT** [comment]
Done.

No worries, I'll wait.

**Comment by PhilT** [comment]
Just to add, the major events are working. If it goes to "no signal" or "emergency calls only" these are reported correctly.

**Comment by demantz** [comment]
Sorry for my very late reply (I was on vacation). This is strange. I looked through the log and updated signal values are showing up in there. Every few minutes there is a updateNotification with new signal strength values. I guess every time the watch screen was activated it updated.

Are you still using the app?

**Comment by PhilT** [comment]
Ah, I was looking for something that continually updates when active. Every
2-5 seconds or so.

My use case is just knowing when I have signal whilst tethered to my laptop
without having to take out my phone.

I'm not using it right now but could give it another go.

Cheers for getting back to me.

**Comment by demantz** [comment]
Ah ok. Well then this is not a bug but more a feature request. Maybe I could add the option to manually select the update interval (right now it updates every time the watch screen turns on - with a delay of 1-2 seconds). But it would be possible to select a periodical update interval. I did not include this option because it will kill the battery if you use a 5 sec interval all day long (bluetooth will always be active and transmitting data)

**Comment by PhilT** [comment]
Yeah, I guess that was my expectation so it seemed like a bug when it
didn't update.

Yeah maybe have it timeout after 30 minutes with this option. I always keep
my watch and phone with Bluetooth on. It'll easily last a day but agree
that transmitting data constantly.

I thought it sent notifications when it lost signal or changed to a
diffident speed but just didn't seem to work the way I would have liked.

I'll give it another test and then try to better articulate how I think it
might be useful for when you're tethered and checking whether you should be
able to load a webpage.

Thanks again for looking into this.
